endpoint.ts

```diff
@@ -52,8 +52,16 @@ export default async (req: Request) => {
       "entrypoint.tsx": `
       import UserCode from "./UserCode.tsx"
 
+      const hasBoard = UserCode.toString().includes("<board")
```

Comment:
i'm surprised this works because this is isn't a valid way to check the user code, `UserCode` is an evaluated React component at this point, it cannot be stringified to the original JSX/typescript code. SOmething like this might work

```suggestion
      const hasBoard = ${userCode.includes("<board").toString()}
```